cmd/snap/cmd_snap_op.go

```diff
@@ -202,18 +203,24 @@ func refreshAll() error {
 	if err != nil {
 		return fmt.Errorf("cannot list updates: %s", err)
 	}
+	// nothing to update/list
+	if len(updates) == 0 {
+		return nil
```

Comment:
@pedronis That's a good idea, and indeed they should go to stderr. There was a longish thread on the change out of the original "snap list" behavior to prevent displaying on stdout output that is broken from the perspective of someone used to the typical output. We should to fix other cases if we find them.

Here a suggested path:

1. Fix "snap list" to send output into Stderr.
2. On this branch, display "All snaps up-to-date." on Stderr (exit 0 still, nothing on stdout).
3. Here or on another branch, make the non-erroneous Stderr output conditional on os.Stderr being a tty.